docs/introduction/get-started/p2p-public.md

```diff
@@ -52,18 +52,18 @@ To do this, we use two terminals with their own state (using their own `miden-cl
 
 ## Transfer assets between accounts
 
-1. Now we can transfer some of the tokens we received from the faucet to our new account B. 
+1. Now we can transfer some of the tokens we received from the faucet to our new account C. 
 
     To do this, from the first client run:
 
     ```shell
-    miden-client tx new p2id <basic-account-id-A> <basic-account-id-B> <faucet-account-id> 50 --note-type public
+    miden-client tx new p2id <basic-account-id-A> <basic-account-id-C> <faucet-account-id> 50 --note-type public
     ```
 
     !!! note
-        The faucet account id can be found on the [Miden faucet website](https://testnet.miden.io/) under the title **Miden faucet**.
+        The faucet account id is `0xad904b3138d71d3e` and can also be found on the [Miden faucet website](https://ethdenver.polygonmiden.io/) under the title **Miden faucet**.
```

Comment:
```suggestion
        The faucet account id is `0xad904b3138d71d3e` and can also be found on the [Miden faucet website](https://testnet.miden.io/) under the title **Miden faucet**.
```

Reply:
are you sure the address is going to stay the same?

Reply:
For now, we will make it more generalized in the future. Now I only want quick feedback